### PR TITLE
PnP Runtime - Regenerated 2020-09-30 SDK for IotHub Gateway Service API

### PR DESCRIPTION
**Pull request by c-ryan-k**
---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to the IoT extension!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] If introducing new functionality or modified behavior, are they backed by unit and integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** unit **and** integration tests passed locally? i.e. `pytest <project root> -vv`
- [ ] Have static checks passed using the .pylintrc and .flake8 rules? Look at the CI scripts for example usage.
- [ ] Have you made an entry in HISTORY.rst which concisely explains your feature or change?

### CODE REVIEW

**Comment by c-ryan-k** [comment]
Unit tests are good
image

Ran into one failure (likely a service hiccup) in monitor events, DT tests are returning 507s so may need to try these under a different subscription:
image

File: azext_iot/sdk/iothub/pnp_runtime/version.py [hunk 1]
```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "2020-06-30-preview"
+VERSION = "2020-09-30"
```
Comment (digimaun) [inline azext_iot/sdk/iothub/pnp_runtime/version.py:12]: This is a future dated API version. Is it already GA'd?

Comment (digimaun) [reply azext_iot/sdk/iothub/pnp_runtime/version.py:12]: Asking this to ensure if we have to make a release before then, we don't break users using the PnP runtime commands.

Comment (c-ryan-k) [reply azext_iot/sdk/iothub/pnp_runtime/version.py:12]: I've been told this version is indeed GA'd - reaching out to service team for final validation and will update once I have it.

Comment (c-ryan-k) [reply azext_iot/sdk/iothub/pnp_runtime/version.py:12]: Update - tested successfully with the node sdk pnp device sample for `twin show`, `twin update`, and `invoke-command`

Comment (digimaun) [reply azext_iot/sdk/iothub/pnp_runtime/version.py:12]: OK great. Could you please also add an entry to HISTORY.rst under a new `0.11.0` version.

**Review by digimaun — APPROVED** [review]

**Pull request merged**